src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.suppliers;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        return Color.values()[random.nextInt(Color.values().length - 1)].getColor();

```

Comment:
you don't need to substact `1` from the array length, since parameter in `nextInt()` method is exclusive

take a look at `nextInt(int bound)` javadoc:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/85908570/2cb29a24-a503-4c9e-abf5-f7a29e8c6ec1)